helm/values.yaml

```diff
@@ -46,36 +42,6 @@ image:
   repository: exporter-trigger
   tag: 'latest'
 
-opa:
-  image: openpolicyagent/opa
-  imageTag: 0.29.4-envoy-3
-  port: 9191
-  enableLogging: false
-  resources:
-    enabled: true
-    value:
-      limits:
-        cpu: 100m
-        memory: 128Mi
-      requests:
-        cpu: 100m
-        memory: 128Mi
-envoy:
-  image: envoyproxy/envoy
-  imageTag: v1.20.7
-  port: 8000
-  maxRequestBytes: 8192
-  requestedDomain: rasterExport
-  resources:
-    enabled: true
-    value:
-      limits:
-        cpu: 100m
-        memory: 128Mi
-      requests:
-        cpu: 100m
-        memory: 128Mi
-
 env:
   port: 8080
   internalPort: 80
```

Comment:
what is the use of "internalPort"?

Reply:
currently there is no use for this parameter, i think someone wanted to implement support of no-auth route but right now it is not implemented